include/circt/Dialect/MSFT/MSFTOps.td

```diff
@@ -7,24 +7,25 @@
 //===----------------------------------------------------------------------===//
 
 include "circt/Dialect/HW/HWOpInterfaces.td"
+include "circt/Dialect/HW/HWTypes.td"
 
 def InstanceOp : MSFTOp<"instance", [
-        Symbol,
         DeclareOpInterfaceMethods<OpAsmOpInterface, ["getAsmResultNames"]>,
+        DeclareOpInterfaceMethods<InnerSymbol, ["getTargetResultIndex"]>,
         DeclareOpInterfaceMethods<SymbolUserOpInterface>,
         DeclareOpInterfaceMethods<HWInstanceLike>
     ]> {
   let summary = "Instantiate a module";
 
-  let arguments = (ins SymbolNameAttr:$sym_name,
+  let arguments = (ins InnerSymAttr:$inner_sym,
```

Comment:
Nit: Should this also be `innerSym`?

Reply:
I'm not sure if every agrees with my logic here, but I used `inner_sym` as the attribute name, and `innerSym` when it is the parameter for a c++ function.  So the builders all take `innerSym`, otherwise clang-tidy will warn about either a) `inner_sym` is the wrong style, or b) the declaration's argument name does not match the definition's.  We don't seem to get CI failures for B but it is enabled in our clang tidy with `readability-inconsistent-declaration-parameter-name`.  I think (guess, really) the reason CI doesn't complain is because the declaration ends up in a generated file, and it gets filtered out since its not in the committed lines :).

Reply:
`inner_name` is consistent with `sym_name` (dictated by upstream), so I'm OK with this reasoning. The fact that upstream is mixing snake_case with camelCase isn't ideal.